doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+SYSBAL(dir) | MA | pgm dir | Define directory for storing and retrieving generated BAL files.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSCPY(dir) | MA | pgm dir | Define one or more directories for retrieving CPY source input files.  If the option starts with + the directories listed will be concatenated with current list.  Multiple directories are always separated by +.  This option may also override suffix by adding *.sfx.
+SYSDAT(dir) | M | pgm dir | Define directory for retrieving DAT source input files for AREAD.  Set to SYSDAT(.) for current directory when no path is specified on DSNAME parm for AREAD.
+SYSERR(dir) | MALE | pgm dir | Define directory for output ERR files. This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSLKD(file) | L | pgm dir | Define input source file for linker commands such as INCLUDE, ALIAS, ENTRY, and NAME.
+SYSLOG(dir) | E | pgm dir | Define directory for output LOG file. This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSLST(dir) | AL | pgm dir | Define directory for output LST files.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
+SYSMAC(dir) | M | mac dir | Define one or more directories for source MAC files.  If the option starts with + the directories listed will be appended to the maclib list.  Multiple directories can be specified in one parameter and are always separated by +. When you specify this option more than once and want previous parameter instructions to remain in place, ensure that you start parameter with a `+` otherwise it will drop prior directories.  This option may also override suffix by adding *.sfx.
+SYSMLC(dir) | MA | pgm dir | Define directory for source input MLC files.
+SYSOBJ(dir) | AL | pgm dir | Define directory for OBJ relocatible object files
+SYSOPT(dir) | MALE | pgm dir | Define directory for @file option files.  This option may override suffix using *.sfx.
+SYSPARM('text') | M | none | Define text string which can be accessed by mz390 global macro variable &SYSPARM.
+SYSPCH(dir) | M | pgm dir | Define directory for PCH output files from PUNCH.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).  Note PUNCH output is in ASCII source format and is not directed to the OBJ file.  These files may be used as linker input commands via SYSLKD(file) option.  Set to SYSPCH(.) for current directory when no path is specified on DSNAME parm for PUNCH.
+SYSPRN(dir) | A | pgm dir | Define directory for assembler listing PRN output files.  This option may also override file name and suffix.  This option may also override file name and/or suffix (*.sfx overrides just the suffix in the pgm directory).
```

Comment:
Line 101. Delete second sentence "This option ... and suffix.". It is a duplicate of the sentence that follows it.

Reply:
Fixed 40c73aa